package.json

```diff
@@ -87,6 +87,7 @@
     "@babel/runtime": "7.17.9",
     "@commitlint/cli": "12.1.4",
     "@commitlint/config-conventional": "12.1.4",
+    "@non-babel/verdaccio-auth-custom-test": "0.0.1",
```

Comment:
Just dummy package in order to test scoped packages on test, name is weird but :) npmjs din't allow me to publish due spammy 🙄 name.